thumb.php

```diff
@@ -770,7 +770,7 @@ protected function processImageAndWriteToCache($localImage)
         }
 
         // create a new true color image
-        $canvas = imagecreatetruecolor($newWidth, $newHeight);
+        $canvas = imagecreatetruecolor(intval($newWidth), intval($newHeight));
```

Comment:
For consistency, it would better to use only (int) instead of intval.
Plus (int) is faster :)